pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -142,26 +142,26 @@ self: super: {
   vector = dontCheck (if pkgs.stdenv.isi686 then appendConfigureFlag super.vector "--ghc-options=-msse2" else super.vector);
 
   conduit-extra = if pkgs.stdenv.isDarwin
-    then super.conduit-extra.overrideAttrs (drv: { __darwinAllowLocalNetworking = true; })
-    else super.conduit-extra;
+                  then super.conduit-extra.overrideAttrs (drv: { __darwinAllowLocalNetworking = true; })
```

Comment:
Could you please explain a bit what this addition is about? Why is that attribute required?